tests/training/test_predictor.py

```diff
@@ -0,0 +1,37 @@
+from tests.utils import check_for_no_backend
```

Comment:
Didn't work locally on may machine. The other tests are using a relative import, maybe change this to be consistent.